07/fundamentals.html

```diff
@@ -268,82 +268,80 @@ <h2 id="flushing"><a href="#flushing">Message queueing, chunks and flushing</a><
-    | teeworlds packet header | chunk header  | motd  | chunk header  | server settings | chunk header  | CON_READY |
-    |        chumks = 3       |     vital     |       |     vital     |                 | vital &amp; flush |           |
-    +-------------------------+---------------+-------+---------------+-----------------+---------------+-----------+
+    +-------------------------+---------------+------+---------------+-----------------+---------------+-----------+
+    | teeworlds packet header | chunk header  | motd | chunk header  | server settings | chunk header  | CON_READY |
+    |        chunks = 3       |     vital     |      |     vital     |                 | vital &amp; flush |           |
+    +-------------------------+---------------+------+---------------+-----------------+---------------+-----------+
             </code>
         </pre>
         <hr>
         <h2 id="int_packing"><a href="#int_packing">Int packing</a></h2>
         <p>
-            Note there is also documentation on ints in
-            <a href="https://github.com/heinrich5991/libtw2/blob/bb63d4d6344a2330ba22d10d638b2cf0d8959544/doc/int.md">
-                the libtw2 docs.
-            </a>
-            <br>
-            There is also a more
-            <a href="../shared/int_packing.html">
-                technical section
-            </a>
-            with examples about int packing.
-            <br>
-            <br>
-            All numbers (integers) sent via the network are packed using a custom
-            teeworlds specific packer.
-            <br>
-            Eventho it being a custom packer the numbers 0-63
-            are pretty much the standard binary representation.
-            <br>
-            <br>
-            So binary <code>00000001</code> is decimal <code>1</code><br>
-            and binary <code>00000010</code> is decimal <code>2</code><br>
-            and so on..
-            <br>
-            <br>
+            <blockquote>
+                <a href="https://github.com/heinrich5991/libtw2/blob/bb63d4d6344a2330ba22d10d638b2cf0d8959544/doc/int.md">
+                    The libtw2 docs
+                </a>
+                also has documentation on ints.
+                <br>
+                There is also a more
+                <a href="../shared/int_packing.html">
+                    technical overview
+                </a>
+                with examples about int packing.
+            </blockquote>
+            <br>
+            All integer numbers sent through the network are packed using a custom
+            Teeworlds-specific packer.
+            <br>
+            Even though it's a custom packer, the numbers 0-63
+            are pretty much the standard binary representation;
+            binary <code>00000001</code> is decimal <code>1</code>,
+            binary <code>00000010</code> is decimal <code>2</code>, and so on.
```

Comment:
I liked the old alignment of the numbers better. It may have been unconventional english but the structure is easier to read for me.

![image](https://github.com/ChillerDragon/teeworlds-protocol/assets/20344300/31051b49-1846-4b19-8564-cfc3362ad855)

Reply:
Alright makes sense, I'll change it back